### PR TITLE
Feature/docker build

### PR DESCRIPTION
**Pull request by askoufis**
Added some docker build scripts, added example commands to the README and I bumped the LTS version to 7.3 because why not.

Added a `-v` flag so it would work with the docker build. Unfortunately, since your ascii art prints every time the program is run, I had to move it to print only after the command line args are parsed, because the dinosaur was sneaking into the version number.

### CODE REVIEW

**Comment by tmortiboy** [comment]
037cdaa172fab4ef89e19004ce18113c

File: build-publish-docker.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
+
+# Builds and publishes the container. We use the ai-admin-api-build
```
Comment (daniel-chambers) [inline build-publish-docker.sh]: `ai-admin-api-build`?

File: build-publish.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+VERSION=$("$BINARY" --version | sed 's/^Version\s\+\([[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]\+\)$/\1/')
+echo "Setting build version to: $VERSION"
+echo "##teamcity[buildNumber '$VERSION']"
+
+# need to ensure we have the Linux binary
+if [[ $BINARY != *"x86_64-linux"* || ! -f $BINARY ]]; then
+  >&2 echo "Linux binary not found. Try running ./build-publish-docker.sh"
+  exit 1
+fi
```
Comment (daniel-chambers) [inline build-publish.sh]: I know this is copy and paste, but let's move the check to see if the exe exists above the lines that use it (ie. the VERSION= lines :)

(Would appreciate it if you could fix it in the place that you got this from too!)

**Pull request merged**